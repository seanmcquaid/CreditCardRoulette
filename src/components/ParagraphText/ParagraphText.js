import React from "react";
import {Text} from "react-native";
import styled from "styled-components/native";

export default styled(Text)`
    font-family : Merriweather-Regular;
    font-size : 12px;
    margin : 6px;
`;