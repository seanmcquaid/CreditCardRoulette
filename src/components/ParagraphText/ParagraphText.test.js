import React from "react";
import {render} from "react-native-testing-library";
import ParagraphText from "./ParagraphText";

describe("<ParagraphText/>", () => {
    it("Matches snapshot correctly", () => {
        const paragraphText = render(<ParagraphText>Paragraph Text</ParagraphText>);
        expect(paragraphText).toMatchSnapshot();
    });
});