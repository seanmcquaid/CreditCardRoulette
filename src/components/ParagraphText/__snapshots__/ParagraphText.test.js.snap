// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ParagraphText/> Matches snapshot correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "fontFamily": "Merriweather-Regular",
        "fontSize": 16,
        "marginBottom": 6,
        "marginLeft": 6,
        "marginRight": 6,
        "marginTop": 6,
      },
    ]
  }
>
  Paragraph Text
</Text>
`;
